pwiz_tools/Skyline/Model/Databinding/Entities/ColumnToolTips.resx

```diff
@@ -1,479 +1,497 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-	<!-- 
-		Microsoft ResX Schema
+  <!-- 

```

Comment:
Again, too much changing here. Please limit the diff to the things you actually want to change.

Reply:
The last commit reverted all the unnecessary changes, it's much more compact now.